doc/sfsmount.1.adoc

```diff
@@ -157,10 +157,7 @@ operation in milliseconds (default: 2000).
 
 *-o cacheexpirationtime=*'MSEC'::
 Set timeout for read cache entries to be considered valid in milliseconds. 0
-disables cache (default: 0). Due to a bug, this can sometimes result in invalid
-data being returned if reading the mount point from multiple processes/threads
-and it's not disabled. Verify data being sent using a checksum if using this
-feature.
+disables cache (default: 0).
```

Comment:
Default is not 0 anymore in these commits

Reply:
Humm, it seems that before your commit disabling the read cache by default it was in 0, cuz I reverted it.

Nice catch :)